### ISSUE TITLE
x86/generator: Could not find an instruction to patch flags

### ISSUE THREAD

**Issue by AssafKlein-tech** [microsoft/sca-fuzzer#64]
I tried fuzzing using this configuration file:

```yaml
supported_categories:
  - BASE-COND_BR
  - BASE-BITBYTE
  - BASE-CMOV
  - BASE-FLAGOP
  - BASE-LOGICAL

contract_observation_clause: ct
contract_execution_clause:
   - seq

min_bb_per_function: 2
max_bb_per_function: 2
test_case_size: 24
avg_mem_accesses: 12
program_generator_seed: 0
input_gen_entropy_bits: 16
memory_access_zeroed_bits: 0
```

 and this command:

 
```bash
./cli.py fuzz -s x86/isa_spec/base.json -i 35 -n 500 -c tests/
```

After a few hunters of rounds, I encounter this exception. Every time it happens in a different round.

```bash
generator.GeneratorException: Could not find an instruction to patch flags ['OF'] , when OF modification in the supported instructions
```

The instructions I enable implicitly set the OF flag, but still this exception is raised.

**Comment by OleksiiOleksenko** [comment]
Could you post some details on the issue? Like the config file that you used

**Comment by AssafKlein-tech** [comment]
I tried fuzzing using this configuration file with a random seed (0).
./cli.py fuzz -s x86/isa_spec/base.json -i 35 -n 500 -c tests/<configuration file>

After a few hunters of rounds, I encounter this exception. Every time it happens in a different round.
The instructions I enable implicitly set the OF flag, but still this exception is raised.

[configuration.txt](https://github.com/microsoft/sca-fuzzer/files/11201298/configuration.txt)

**Comment by OleksiiOleksenko** [comment]
Updated the bug report based on the info you sent. The gem5 bits are irrelevant for this issue, hence removed from the description.

**Comment by OleksiiOleksenko** [comment]
Closed by 8102ded
